process.md

```diff
@@ -82,24 +88,14 @@ When packaging your artifact, please keep in mind: a) how accessible you are mak
 
 A good way to package artifacts is as a virtual machine (VM). VMs give an easily reproducible environment that is somewhat resistant to bit rot. They also give reviewers confidence that errors or other problems cannot cause harm to their machines. Source code artifacts should use Docker or another build tool to manage all compilation and dependencies. This improves the odds that the reviewers will be able to quickly and painlessly install the artifact — without getting lost in environment issues ([e.g. what Python do I need?!](https://xkcd.com/1987/)).
 
-Submit your artifact as a single archive file (e.g., zip).
-
 # Support
 
-## Informative session at CI2024
-
-There will be a 1 hour informal session for authors who are considering submitting artefacts to give an overview of the approach, some lightning talks on producing reproducible artefacts, discussion of the criteria, and an opportunity to ask questions about the process.
-
-# Other notes
+There will be a 1-hour informal session for authors ==and reviewers== who are considering submitting artefacts to give an overview of the approach. This will happen online after CI 2024.
```

Comment:
```suggestion
There will be a 1-hour informal session for authors ==and reviewers== who are considering submitting artefacts to give an overview of the approach. This will happen online after CI2024 and informed via the dedicated [webpage for AE](https://alan-turing-institute.github.io/climate-informatics-2024/artefact-evaluation/) in the conference website.
```